amazoncorretto-11-debian-slim/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM debian:stable-slim
+
+# install corretto
+RUN apt-get update \
+  && apt-get dist-upgrade -y \
```

Comment:
I don't think `dist-upgrade` is desirable as the parent image is supposedly up to date all the time

Reply:
Removed the `dist-upgrade` & also kept `gnupg` installed as asserted by the tests.